packages/react-widgets/src/models/fqn.d.ts

```diff
@@ -0,0 +1,69 @@
+import { Provider } from "@carto/react-core"
```

Comment:
All the code has been copied from https://github.com/CartoDB/cloud-native/blob/main/commons/lib/fqn.ts
The idea is to move it to an external module at some point.